x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -1657,6 +1659,90 @@ export class EndpointDocGenerator extends BaseDataGenerator {
     };
   }
 
+  /**
+   * Generate a Fleet EPM Package for Endpoint
+   */
+  public generateEpmPackageInfo(): GetInfoResponse['item'] {
```

Comment:
I agree. I initially tried updating `generateEpmPackage` but it adds a bunch of additional file touches since the returned type is different. I think it's best to have a separate PR for removing `/api/fleet/epm/packages?category=security` related code from endpoint since this PR has already gotten quite large.